src/StackHCI/custom/stackhci.ps1

```diff
@@ -364,9 +364,14 @@ Function Print-FunctionParameters{
         if ([System.Management.Automation.PSCmdlet]::CommonParameters -contains $param.key) {
             continue
         } 
-        if ($param.key -in @("ArmAccessToken","ArcSpnCredential","Credential","AccountId","GraphAccessToken")) { continue } 
-
-        $body.add($param.Key, $param.Value)
+        if ($param.key -in @("ArmAccessToken","ArcSpnCredential","Credential","AccountId","GraphAccessToken","AccessToken")) 
+        {
+            $body.add($param.Key, "XXXXXXX") 
```

Comment:
What does "XXXXXX" means?

Reply:
This code logs the parameters. We are taking credentials out and logging them as XXXXX.

Reply:
Then I suggest to define "XXXXXX" as an variable with name or add a comment here.

Reply:
Updated the code as per suggestion. thanks